.bazelrc

```diff
@@ -86,7 +86,10 @@ build:remote --remote_accept_cached=true
 ################################
 
 # Releases should always be stamped with version control info
-build:release --workspace_status_command="yarn ng-dev release build-env-stamp --mode=release"
+# Note: We usually would use the `ng-dev` tool directly, but that one would need to be
+# built with Bazel and building with Bazel inside the Bazel-invoked workspace status
+# cript does not work. To workaround this we have a `ts-node`-based stamping script.
+build:release --workspace_status_command="yarn build-env-stamp --mode=release"
```

Comment:
Can we have a TODO that specifically notes that we would like to not have to have this workaround?

Reply:
Added. I don't think there is a way to achieve this without any weird tricks, but good to have it on the radar 👍